### PR TITLE
Release v2.1.9

### PR DESCRIPTION
**Pull request by kozlovic**
Signed-off-by: Ivan Kozlovic <ivan@synadia.com>

### CODE REVIEW

File: 2.1.9/scratch/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM scratch
-COPY --from=nats:2.1.8-alpine3.11 /usr/local/bin/nats-server /nats-server
+COPY --from=nats:2.1.9-alpine3.11 /usr/local/bin/nats-server /nats-server
```
Comment (derekcollison) [inline 2.1.9/scratch/Dockerfile:2]: Alpine latest is 3.12

File: tests/build-images.sh [hunk 2]
```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env bash
 set -ex
 
-ver=(NATS_SERVER 2.1.8)
+ver=(NATS_SERVER 2.1.9)
 
 (
 	cd "${ver[1]}/alpine3.11"
-	docker build --tag nats:2.1.8-alpine3.11 .
+	docker build --tag nats:2.1.9-alpine3.11 .
```
Comment (derekcollison) [inline tests/build-images.sh:8]: Upgrade to 3.12

File: tests/run-images.sh [hunk 3]
```diff
@@ -2,8 +2,8 @@
 set -ex
 
 images=(
-	'nats:2.1.8-alpine3.11'
-	'nats:2.1.8-scratch'
+	'nats:2.1.9-alpine3.11'
```
Comment (derekcollison) [inline tests/run-images.sh:5]: Same here.

**Review by derekcollison — CHANGES_REQUESTED** [review]

**Review by derekcollison — APPROVED** [review]
LGTM

**Review by wallyqs — APPROVED** [review]
LGTM

**Pull request merged**